compiler/src/iree/compiler/Dialect/Modules/VMVX/Conversion/VMVXToVM/test/add.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: iree-opt --split-input-file --iree-vm-conversion --canonicalize %s | FileCheck %s
+
+// CHECK-LABEL: @add_2d_f32
+func.func @add_2d_f32(
+    // LHS
+    %arg0 : memref<128xf32>, %arg1 : index, %arg2 : index, %arg3 : index,
+    // RHS
+    %arg4 : memref<65536xf32>, %arg5 : index, %arg6 : index, %arg7 : index,
+    // OUT
+    %arg8 : memref<65536xf32>, %arg9 : index, %arg10 : index, %arg11 : index,
+    // SIZE
+    %arg12 : index, %arg13 : index) {
+
+  //  CHECK-DAG: %[[LHS_OFFSET:.*]] = vm.ext.i32.i64.s %arg1 : i32 -> i64
```

Comment:
(I think) if you add `--iree-vm-target-index-bits=64` to the RUN line you'll get rid of all these exts and be able to have a cleaner test

Reply:
Oh nice - done.